core/modules/views_ui/js/views-admin.js

```diff
@@ -298,120 +298,122 @@ Backdrop.behaviors.viewsUiSearchOptions = {
- * to the user's search term. Typing in "taxonomy" will show only those options
- * containing "taxonomy" in their label.
- */
-Backdrop.viewsUi.OptionsSearch = function ($form) {
-  this.$form = $form;
-  // Add a keyup handler to the search box.
-  this.$searchBox = this.$form.find('#edit-options-search');
-  this.$searchBox.on('keyup', $.proxy(this.handleKeyup, this));
-  // Get a list of option labels and their corresponding divs and maintain it
-  // in memory, so we have as little overhead as possible at keyup time.
-  this.options = this.getOptions(this.$form.find('.filterable-option'));
-  // Restripe on initial loading.
-  this.handleKeyup();
-
-  // Add a change handler to the filter group select.
-  this.$filterGroup = this.$form.find('select[name="group"]');
-  this.$filterGroup.on('change', $.proxy(this.filterGroup, this));
-  // Trap the ENTER key in the search box so that it doesn't submit the form.
-  this.$searchBox.on('keypress', function (event) {
-    if (event.which == 13) {
-      event.preventDefault();
-    }
-  });
-};
-
-$.extend(Backdrop.viewsUi.OptionsSearch.prototype, {
   /**
-   * Assemble a list of all the filterable options on the form.
+   * Constructor for the viewsUi.OptionsSearch object.
    *
-   * @param $allOptions
-   *   A $ object representing the rows of filterable options to be
-   *   shown and hidden depending on the user's search terms.
+   * The OptionsSearch object filters the available options on a form according
+   * to the user's search term. Typing in "taxonomy" will show only those options
+   * containing "taxonomy" in their label.
    */
-  getOptions: function ($allOptions) {
-    var i, $label, $description, $option;
-    var options = [];
-    var length = $allOptions.length;
-    for (i = 0; i < length; i++) {
-      $option = $($allOptions[i]);
-      $label = $option.find('label');
-      $description = $option.find('div.description');
-      options[i] = {
-        // Search on the lowercase version of the label text + description.
-        'searchText': $label.text().toLowerCase() + " " + $description.text().toLowerCase(),
-        // Maintain a reference to the jQuery object for each row, so we don't
-        // have to create a new object inside the performance-sensitive keyup
-        // handler.
-        '$div': $option
-      };
-    }
-    return options;
-  },
+  Backdrop.viewsUi.OptionsSearch = function ($form) {
+    /**
+     *
+     * @type {jQuery}
+     */
+    this.$form = $form;
 
-  /**
-   * Keyup handler for the search box that hides or shows the relevant options.
-   */
-  handleKeyup: function (event) {
-    var found, i, j, option, search, words, wordsLength, zebraClass, zebraCounter;
+    const searchBoxSelector = '#edit-options-search';
+    const controlGroupSelector = 'select[name="group"]';
+    this.$form.on(
+      'formUpdated',
+      `${searchBoxSelector},${controlGroupSelector}`,
```

Comment:
We don't often see backticks in Backdrop JS code, as this kind of string interpolation did not work in any version of IE at all (even up to 11), see https://caniuse.com/template-literals. As we no longer support any IE version, I think we're now safe to include this.